src/SFML/Network/Ftp.cpp

```diff
@@ -544,12 +544,12 @@ Ftp::Response Ftp::DataChannel::open(Ftp::TransferMode mode)
             Uint8 data[6] = {0, 0, 0, 0, 0, 0};
             std::string str = response.getMessage().substr(begin);
             std::size_t index = 0;
-            for (int i = 0; i < 6; ++i)
+            for (unsigned char& datum : data)
```

Comment:
Maybe name it `byte`?

Reply:
Hmm... not entirely against the suggestion, but:
1. C++17 introduces `std::byte`, and it might cause some confusion
2. The container is named `data`. Maybe the container should be named `bytes` instead? If so, I'd rather perform that refactoring in a separate PR.

Let me know if you feel strongly about this.

Reply:
Not very strongly, just thought "datum" is quite unusual. Naming the container `bytes` is also OK. Not sure if a PR just for that isn't too much effort; but maybe you can combine it with other things.